zerver/lib/markdown/api_arguments_table_generator.py

```diff
@@ -91,8 +91,8 @@ def render_table(self, arguments: Sequence[Mapping[str, Any]]) -> List[str]:
         # TODO: Fix naming now that this no longer renders a table.
         table = []
         argument_template = """
-<div class="api-argument">
-    <p class="api-argument-name"><strong>{argument}</strong> {required} {deprecated}</p>
+<div class="api-argument" id="parameter-{argument}">
+    <p class="api-argument-name"><strong>{argument}</strong> {required} {deprecated}<a href="#parameter-{argument}" class="api-argument-hover-link"><i class="fa fa-chain"></i></a></p>
```

Comment:
I also changed the whitespace here to make sure there's a space after `{deprecated}`; otherwise it looks wrong for deprecated parameters!